src/main/java/it/gov/pagopa/pu/pagopapayments/connector/pagopa/gpd/config/GpdApiClientConfig.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.pu.pagopapayments.connector.pagopa.gpd.config;
+
+import it.gov.pagopa.pu.pagopapayments.config.ApiClientConfig;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "rest.pagopa-node-services.gpd")
```

Comment:
add also the application.yml change and link the PR on aks-deployment to fill the new ENV vars